### ISSUE TITLE
[OAKTrust] Community example page

### ISSUE THREAD

**Issue by devangm** [TAMULib/dspace-angular#204]
**URL:** https://oaktrust-dev.library.tamu.edu/communities/0f6b340a-81d7-47a0-9a6d-caf1c9185c2b/search
**WCAG 2.1 AA compliant? :** No
**Violation:** 4.1.2 Name, Role, Value (A)
**Issues:** broken ARIA menu
**Notes:** There seems to be a non-interactable ARIA menu that doesn't need to be there.
